src/test/javascript/e2e/scenarios/discontinue-subject-should-unassign-source.spec.ts

```diff
@@ -120,19 +124,6 @@ describe('Discontinued subject should unassign sources', () => {
         });
     });
 
-    it('should be able to delete a source', function () {
-        element.all(by.linkText(sourceName))
-            .all(by.xpath('ancestor::tr'))
-            .all(by.cssContainingText('button', 'Delete'))
-            .click().then(() => {
-                element(by.css('.modal-footer button.btn-danger')).click().then(() => {
```

Comment:
we should not be able to delete a source that was assigned to subject. Maybe we can change this check to test that?

Reply:
Makes sense!